### PR TITLE
SHIP-2386 golang added npm and publish alpha from feature-branch

### PR DESCRIPTION
**Pull request by sgeisbacher**
- added optional step to publish a `-alpha` version from feature-branch
- added npm as it is need down the road to install sls-plugins

### CODE REVIEW

File: golang-deploy/Dockerfile [hunk 1]
```diff
@@ -3,11 +3,26 @@ ARG DOCKERIZE_VERSION="v0.6.1"
...
 USER root
 RUN apt-get update && \
-    apt-get install -y git-crypt make sshpass awscli tree
+    apt-get install -y \
+        git-crypt \
+        make \
+        sshpass \
+        awscli \
+        tree \
+        nodejs npm
```
Comment (serratus) [inline golang-deploy/Dockerfile:12]:
```suggestion
        nodejs \
        npm
```

**Review by serratus — DISMISSED** [review]
Glad to see the introduction of `alpha` flags 👍  this should make it so much easier to test.

Just to clarify, the `latest` tag is always set to the `alpha` tag too. Is that intended?

File: golang-deploy/Dockerfile (hunk 1 above)
Comment (sgeisbacher) [reply golang-deploy/Dockerfile:12]: that `nodejs npm` was intended in one line cause it is a "group" 🤷‍♂️  but no strong opinion on that 😅

**Comment by sgeisbacher** [comment]
> Just to clarify, the latest tag is always set to the alpha tag too. Is that intended?

@serratus thats a good point 🤔 i would rather remove that push of the `latest` tag. we are and should use a specific version everywhere. cause actually in the current case there was a breaking change with that serverless installation

**Review by serratus — APPROVED** [review]

**Pull request merged**